Standards/scs-03XX-v1-standard-roles.md

```diff
@@ -0,0 +1,271 @@
+
+**Core Roles:**
+
+- member
+- admin
+- service
+
+**Service-specific Roles:**
+
+- manager (Keystone)
+- key-manager:service-admin (Barbican)
+- creator (Barbican)
+- observer (Barbican)
+- audit (Barbican)
+- load-balancer_observer (Octavia)
+- load-balancer_global_observer (Octavia)
+- load-balancer_member (Octavia)
+- load-balancer_quota_admin (Octavia)
+- load-balancer_admin (Octavia)
+- ResellerAdmin (Swift + Ceilometer)
+- heat_stack_user (Heat)
+
+#### Role Definitions
+
+The following overview will explain the roles' purposes and target scopes.
+Roles marked as "internal" are roles only meant to be assigned to technical user accounts intended for internal use by OpenStack services.
+
+Core Roles:
+
+| Role | Primary Target(s) | Purpose |
+|---|---|---|
+| member | customer | read and write access to resources in the scope of authentication (e.g. project) |
+| admin | CSP | cloud-level global administrative access to all resources (cross-domain, cross-project) |
+| service | internal | internal technical user role for service communication |
+
+Service-specific Roles:
+
+| Service | Role | Primary Target(s) | Purpose |
+|---|---|---|---|
+| Keystone | manager | customer, CSP | slightly more elevated privileges than *member*, able to manage core resources or settings of a project or domain; currently this is limited to managing identity resources within a domain |
+| Barbican | audit | customer | allows read-only access to metadata of secrets within a project; does not allow secret retrieval or decryption |
+| Barbican | observer | customer | allows read-only access to secrets within a project, including retrieval and decryption |
+| Barbican | creator | customer | allows access to, creation and deletion of secrets within a project, including retrieval and decryption, equal to the member role |
+| Barbican | key-manager:service-admin | CSP | management API access for the cloud administrator, e.g. for project quota settings |
+| Octavia | load-balancer_observer | customer | access to read-only APIs |
+| Octavia | load-balancer_global_observer | CSP | access to read-only APIs including resources owned by others |
+| Octavia | load-balancer_member | customer | access to read and write APIs |
+| Octavia | load-balancer_quota_admin | CSP | admin access to quota APIs only, including quota resources owned by others |
+| Octavia | load-balancer_admin | CSP | admin access to all LB APIs including resources owned by others |
+| Swift | ResellerAdmin | Ceilometer (internal) | assigned to technical users of Ceilometer to integrate with Swift for access privileges in the object storage API to store statistics for metering |
+| Heat | heat_stack_user | internal | assigned to technical user accounts resulting from other resources' creation in Heat templates |
+
+### API Policies
+
+TODO: what does the CSP need to adhere to when it comes to API policy configuration?
```

Comment:
Maybe we can advise that the policies at least should not be loosened. I don't think we can advise any more than maybe a few edge cases.

Reply:
Right. I added remarks about not permitting any deviation of the default role model in a detrimental manner and addressed the Octavia edge case.